noxfile.py

```diff
@@ -222,13 +224,15 @@ def tests(session: nox.sessions.Session):
     prepare_venv(session)
     session.install("--no-deps", "--editable", ".")
     session.env.update(ENV)
-    session.run(
+    run_args = [
         "python",
         "-m",
         "iris.tests.runner",
         "--default-tests",
-        "--system-tests",
```

Comment:
I've dropped running the system tests because they already get run as part of "default-tests" so there doesn't seem to be much sense in running them twice.